code/processes/chainedtp.q

```diff
@@ -55,7 +55,7 @@ openlog:{[lgfile]
 
 /- subscribe to tickerplant and refresh tickerplant settings
 subscribe:{[]
-  s:.sub.getsubscriptionhandles[`procname;.ctp.tickerplantname;()!()];
+  s:.sub.getsubscriptionhandles[`;.ctp.tickerplantname;()!()];
```

Comment:
I think this change was already merged in the other PR, this might cause a conflict

Reply:
Ah, not sure how that got in there. I'll sort it out